activerecord/lib/active_record/core.rb

```diff
@@ -240,7 +238,7 @@ def filter_attributes
 
       # Specifies columns which shouldn't be exposed while calling +#inspect+.
       def filter_attributes=(attributes_names)
-        @filter_attributes = attributes_names.map(&:to_s).to_set
+        @filter_attributes = attributes_names.to_set
```

Comment:
Now `.to_set` has been useless since `self.class.filter_attributes.include?(attribute_name)` is replaced.

Reply:
I think `inspect` or `pretty_print` is not called in production for typical application.
So I will go like this to defer allocation.

```ruby
      def inspection_filter
        @inspection_filter ||= ActiveSupport::ParameterFilter.new(self.class.filter_attributes, expose: [nil])
      end
```